### PR TITLE
Fixes #24800 - Notify deprecation of taxonomy, login settings

### PR DESCRIPTION
**Pull request by tbrisker**
<strike>Includes 2 commits: first one allows displaying notifications when login is disabled, second actually creates the relevant notification.</strike>

### CODE REVIEW

File: db/seeds.d/170-notification_blueprints.rb
Comment (tbrisker) [inline db/seeds.d/170-notification_blueprints.rb:37]: not directly related, just looks like this was missing since there is no locale context when running seeds.

**Comment by ohadlevy** [comment]
this conflicts with #5612 partly and nearly ready to be merged, can we wait until its merged?

**Comment by tbrisker** [comment]
Split off the notification when login disabled to gh-6040, this doesn't actually depend on it.

**Comment by ekohl** [comment]
This is blocked on https://community.theforeman.org/t/proposal-remove-support-for-disabling-taxonomies-or-login/10972 being accepted. Right now it's still in discussion.

**Comment by tbrisker** [comment]
@ekohl doesn't look like there are any objections on the RFC. Considering that we won't actually be removing any of the code prior to 1.20 branching, we can reconsider if we get pushback from users when 1.20 comes out

**Comment by tbrisker** [comment]
Ping @ekohl?

File: config/application.rb
Comment (ekohl) [inline config/application.rb]: For `init_dynflow` we do this in the `config.after_initialize`. Let's stay consistent and move it there.

**Review by ekohl — DISMISSED** [review]
In general no objections

File: config/application.rb
Comment (tbrisker) [reply config/application.rb]: moved

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
test failure unrealted :green_heart:

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @tbrisker!

**Pull request merged**
